### ISSUE TITLE
Implements layers (0.0.0.20200106)

### ISSUE THREAD

**Issue by takuya-takeuchi** [takuya-takeuchi/NcnnDotNet#10]
* [x] absval.h(AbsVal)
* [x] argmax.h(ArgMax)
* [x] batchnorm.h(BatchNorm)
* [x] bias.h(Bias)
* [x] binaryop.h(BinaryOp)
* [x] bnll.h(BNLL)
* [x] cast.h(Cast)
* [x] clip.h(Clip)
* [x] concat.h(Concat)
* [x] convolution.h(Convolution)
* [x] convolutiondepthwise.h(ConvolutionDepthWise)
* [x] crop.h(Crop)
* [x] deconvolution.h(Deconvolution)
* [x] deconvolutiondepthwise.h(DeconvolutionDepthWise)
* [x] dequantize.h(Dequantize)
* [x] detectionoutput.h(DetectionOutput)
* [x] dropout.h(Dropout)
* [x] eltwise.h(Eltwise)
* [x] elu.h(ELU)
* [x] embed.h(Embed)
* [x] exp.h(Exp)
* [x] expanddims.h(ExpandDims)
* [x] flatten.h(Flatten)
* [x] hardsigmoid.h(HardSigmoid)
* [x] hardswish.h(HardSwish)
* [x] innerproduct.h(InnerProduct)
* [x] input.h(Input)
* [x] instancenorm.h(InstanceNorm)
* [x] interp.h(Interp)
* [x] log.h(Log)
* [x] lrn.h(LRN)
* [x] lstm.h(LSTM)
* [x] memorydata.h(MemoryData)
* [x] mvn.h(MVN)
* [x] noop.h(Noop)
* [x] normalize.h(Normalize)
* [x] packing.h(Packing)
* [x] padding.h(Padding)
* [x] permute.h(Permute)
* [x] pooling.h(Pooling)
* [x] power.h(Power)
* [x] prelu.h(PReLU)
* [x] priorbox.h(PriorBox)
* [x] proposal.h(Proposal)
* [x] psroipooling.h(PSROIPooling)
* [x] quantize.h(Quantize)
* [x] reduction.h(Reduction)
* [x] relu.h(ReLU)
* [x] reorg.h(Reorg)
* [x] requantize.h(Requantize)
* [x] reshape.h(Reshape)
* [x] rnn.h(RNN)
* [x] roialign.h(ROIAlign)
* [x] roipooling.h(ROIPooling)
* [x] scale.h(Scale)
* [x] selu.h(SELU)
* [x] shufflechannel.h(ShuffleChannel)
* [x] sigmoid.h(Sigmoid)
* [x] slice.h(Slice)
* [x] softmax.h(Softmax)
* [x] split.h(Split)
* [x] spp.h(SPP)
* [x] squeeze.h(Squeeze)
* [x] tanh.h(TanH)
* [x] threshold.h(Threshold)
* [x] tile.h(Tile)
* [x] unaryop.h(UnaryOp)
* [x] yolodetectionoutput.h(YoloDetectionOutput)
* [x] yolov3detectionoutput.h(Yolov3DetectionOutput

**Comment by takuya-takeuchi** [comment]
Added new and delete methods
